deploy/charts/cert-manager/templates/cainjector-deployment.yaml

```
@@ -59,7 +59,7 @@ spec:
           {{- end }}
           imagePullPolicy: {{ .Values.cainjector.image.pullPolicy }}
           args:
-          {{- if .Values.global.logLevel }}
+          {{- if eq (.Values.global.logLevel | toString) "0" "1" "2" "3" "4" "5" }}
```

I think the following change fixes the issue without limiting the values for logLevel:
```suggestion
          {{- if not (kindIs "invalid" .Values.global.logLevel) }}
```
based on: https://github.com/Masterminds/sprig/issues/53#issuecomment-483414063

---

fixed in https://github.com/cert-manager/cert-manager/pull/5860#issuecomment-1875103608